t81_558_class_10_2_lstm.ipynb

```diff
@@ -708,7 +708,7 @@
     "model.add(Dense(4, activation='sigmoid'))\n",
     "\n",
     "# try using different optimizers and different optimizer configs\n",
-    "model.compile(loss='binary_crossentropy',\n",
+    "model.compile(loss='categorical_crossentropy',\n",
```

Comment:
It works with binary_crossentropy too but switched this to categorical_crossentropy since this is a classification task